samples/api-doc-search/nodejs/routes/search.js

```diff
@@ -0,0 +1,149 @@
+// <copyright file="search.js" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// server.js is used to setup and configure your API server.
```

Comment:
File name looks incorrect.

Reply:
This is Search functionality based sample for User queries, so added the name.

Reply:
// server.js is used to setup and configure your API server.